src/Makevars

```diff
@@ -1,2 +1,3 @@
 
 CXX_STD = CXX11
+PKG_CPPFLAGS = -mavx -maes
```

Comment:
I suspect we cannot add those unconditionally as they are likely non-portable, possible even betweem `clang` and `gcc`.

Reply:
Yes exactly, this is what I meant with

> meow does require x64 as well as hardware AES, so I assume it's not a good fit for digest over all.